chart/templates/post-delete-hook-ServiceAccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.postDelete.enabled }}
+apiVersion: v1
```

Comment:
File naming shouldn't be capitalized and have dashes. We should probably just stick to dashes for all of these files for style.

Reply:
yes, I followed the pattern of existing files. But it is a good time to switch to dashes :)